starkinfra/creditnotepreview/__creditnotepreview.py

```diff
@@ -0,0 +1,78 @@
+from starkcore.utils.subresource import SubResource
+from starkinfra.utils import rest
+from ..creditnote.invoice.__invoice import Invoice, _resource as _invoice_resource
+from starkcore.utils.api import from_api_json
+
+
+class CreditNotePreview(SubResource):
+    """# CreditNotePreview object
+    CreditNotePreview are used to preview CCB contracts between you and your customers with
+    selected table type.
+    When you initialize a CreditNotePreview, the entity will not be automatically
+    send in the Stark Infra API. The 'create' function sends the objects
+    to the Stark Infra API and returns the list of preview data.
+    ## Parameters (required):
+    - type [string]: The type of the table. ex: "sac", "price", "american", "bullet", "custom"
```

Comment:
- type [string]: table type that defines the amortization system. Options: "sac", "price", "american", "bullet", "custom"